lock.json

```diff
@@ -1452,13 +1452,8 @@
           "artifacts": [
             {
               "algorithm": "sha256",
-              "hash": "21c3f448d17224246e36a5db678e98b9261a1909604c160cd12f45fa1e7eb695",
-              "url": "https://files.pythonhosted.org/packages/6e/f4/7dec21039c12c231508dc06a42efef587b6fda809d330053005b093e84d8/sphinx_library-1.1.2-py3-none-any.whl"
```

Comment:
I published the doc site to get out the providers page from #28 and noticed the missing logo: https://science.scie.app/providers.html